pex/interpreter.py

```diff
@@ -227,12 +241,34 @@ def get(cls, binary=None):
             configured_macosx_deployment_target=configured_macosx_deployment_target,
         )
 
+    # Increment this integer version number when changing the encode / decode format or content.
+    _FORMAT_VERSION = 1
```

Comment:
Wouldn't it make sense to just tie this to the current pex version?

Reply:
I could, but at the expense of thrashing caches for no reason. The INTERP-INFO format changes rarely relative to Pex releases.